app/scripts/controllers/chapter.js

```diff
@@ -118,12 +116,12 @@ angular.module('gdgxHubApp')
     };
 
     $scope.changeCalendarView = function (view, calendar) {
-      calendar.fullCalendar('changeView', view);
+      uiCalendarConfig.calendars[calendar].fullCalendar('changeView', view);
     };
 
-    $scope.events = function (start, end, callback) {
+    $scope.events = function (start, end, timezone, callback) {
```

Comment:
It's not, but it's part of the new fullcalendar API. So it has to be an argument.